README.md

```diff
@@ -0,0 +1,232 @@
+
+<!-- WALKTHROUGH -->
+
+## 🤯 Walk through video <a name="walk-through"></a>
+
+
+- [here](https://www.loom.com/share/e6adbb08d0904cb9ae9034753afeed20)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://budget-app-1n0j.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+- Clone this repo as described in the setup section. 
+- Make modifications as peferred 
+
+
+### Prerequisites
+
+In order to run this project you need: To clone or fork and run it in a browser
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/waelelsafty07/budget-app.git
+
+using Ubuntu:
+
+```sh
+  cd my-desired-folder
+  git clone https://github.com/waelelsafty07/budget-app.git
+```
+
+For more information on how to clone or fork a repository:
+- <a href="https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository">How to clone a repo</a>
+- <a href="https://docs.github.com/en/get-started/quickstart/fork-a-repo">How to fork a repo</a>
+
+### Install
+
+- Run ` npm install ` for linters
+- Run ` bundle install `
+
+### Usage
+
+To run the project, execute the following command:
+
+- Run ` bundle exec rails server`
+
+### Run tests
+
+To run tests, run the following command:
+
+
+- to test CSS linter errors `npx stylelint "**/*.{css,scss}"`
+
+- to test ruby linter errors  `rubocop`
+
+- to test the classes ` rails c`
+
+- to run spec file ` rspec spec` or ` bundle exec rspec spec`
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Wael Elsafty**
+
+
+- GitHub: [@waelelsafty](https://github.com/waelelsafty07)
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+Future changes:
+- Add animations and transitions
+- Add more styles and colors
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/waelelsafty07/budget-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<a href="https://www.flaticon.com/free-icons/recipe" title="recipe icons">Recipe icons created by justicon - Flaticon</a><br />
+
+- Thanks to all Microverse community
```

Comment:
- [ Required ] I think it would be nice if you give credits to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/). Currently, there is no credit to the original author of this design on your readme file, to use the design professionally you are required to give credits to the author. Kindly rectify this.